src/components/chatbox/chatbox.tsx

```diff
@@ -344,12 +342,14 @@ const ChatBox = () => {
 
 			<form onSubmit={handleSubmit} ref={formRef}>
 				<div className={s.bottom}>
-					{textContentScrollBarIsVisible ? (
-						<div className={s.arrowdown} onClick={handleArrowDownClick}>
-							<IconTile size="small">
-								<IconArrowDownCircle16 />
-							</IconTile>
-						</div>
+					{textContentScrollBarIsVisible && !isLoading ? (
+						<Button
+							className={s.arrowdown}
+							icon={<IconArrowDownCircle16 />}
+							onClick={handleArrowDownClick}
+							aria-label="Scroll to bottom"
+							color="secondary"
+						/>
```

Comment:
The swap to `<Button />` addresses 
[✓ Global: scroll tool - missing hover](https://app.asana.com/0/0/1205523765248038/f)
- leverages ready-mage a11y behavior & styles